src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -489,6 +489,38 @@ static IEnumerable<FunctionOverload> GetBicepFilePermittedOverloads()
                     .WithRequiredParameter("resourceType", LanguageConstants.String, "The type of resource within the specified namespace")
                     .Build();
 
+                yield return new FunctionOverloadBuilder("toLogicalZone")
+                    .WithReturnType(LanguageConstants.String)
+                    .WithGenericDescription("Returns the logical zone corresponding to the given physical zone.")
+                    .WithRequiredParameter("subscriptionId", LanguageConstants.String, "The subscription ID of the deployed availability zones")
+                    .WithRequiredParameter("location", LanguageConstants.String, "The location of the availability zone mappings")
+                    .WithRequiredParameter("physicalZone", LanguageConstants.String, "The physical zone to convert")
+                    .Build();
```

Comment:
>Build [](http://example.com/codeflow?start=21&length=5)

Do we need to add `.WithFlags(FunctionFlags.RequiresInlining)` on each function? IIRC the az functions only work in outputs and in the resource body in the JSON.